docs/pages/services/indexer.mdx

```diff
@@ -20,38 +20,150 @@ Having such a database allows clients to quickly and efficiently query the oncha
+| ------------------------- | --------------- | -------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| Needed                    | RPC_HTTP_URL    | The URL to access the blockchain using HTTP                                                              | http://host.docker.internal:8545 (when running in Docker) |
+| Optional                  | RPC_WS_URL      | The URL to access the blockchain using WebSocket                                                         |
+| Optional                  | START_BLOCK     | The block to start indexing from. The block in which the `World` contract was deployed is a good choice. | 1                                                         |
+| Optional                  | DEBUG=mud:\*    | Turn on debugging                                                                                        |                                                           |
+| Optional, only for SQLite | SQLITE_FILENAME | Name of database                                                                                         | `indexer.db`                                              |
+| Required for PostgreSQL   | DATABASE_URL    | URL for the database, of the form `postgres://<host>/<database>`                                         |
 
-### Running directly
+### Running directly with SQLite
 
-To run the indexer directly on your computer:
-
-1. Build the MUD repository, [as per the directions](/contribute).
+To run the indexer directly on your computer using SQLite:
 
 1. Start a `World` to index.
    An easy way to do this is to [use a TypeScript template](/templates/typescript/getting-started) in a separate command line window.
 
-1. Run the indexer from the build MUD repository.
-   For example, to start the indexer with SQLite, use these commands.
+1. Install the indexer.
 
    ```bash copy
-   cd packages/store-indexer
-   export RPC_HTTP_URL=http://localhost:8545
-   pnpm start:sqlite
+   pnpm install @latticexyz/store-indexer@next
    ```
 
-   Alternatively, you can start the indexer for the local blockchain using `pnpm start:sqlite:local`, in which case you don't need to specify `RPC_HTTP_URL`.
+1. Set `RPC_HTTP_URL` and run the indexer.
+
+   ```bash copy
+   export RPC_HTTP_URL=http://127.0.0.1:8545
+   pnpm sqlite-indexer
+   ```
 
-**Note:** The `anvil.db` is persistent if you stop and restart the indexer.
+**Note:** The `indexer.db` is persistent if you stop and restart the indexer.
 If that is not the desired behavior (for example, because you restarted the blockchain itself), delete it before starting the indexer.
 
+### Running directly with PostgreSQL storage
+
+This is the method to use PostgreSQL as merely a data storage solution.
+It has better performance, but the format of the data in PostgreSQL is opaque, you have to use our front end to understand the data.
```

Comment:
```suggestion
The schemaless indexer stores MUD table records into a single monolithic table.
This allows it to index the data of all tables of all MUD Worlds on a chain efficiently, but querying for data is limited to filters on `address`, `tableId`, and the record's key.
Since the record's `data` is stored as encoded bytes blob, it's harder to query based on it in SQL.
The main purpose of this variant of the indexer is to efficiently hydrate a MUD client, which decodes the data.
```